mandatory/1-fix-functions.js

```diff
@@ -10,30 +10,28 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
+function mood(isHappy) {
+  //let isHappy = true;
   if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
+//let num = 12;
+let isBigEnough = num > 10;
```

Comment:
This line should be inside the function **greaterThan10**. Then it will pass without error:
_let isBigEnough = num > 10;_

Reply:
Hi FatemRahimi. Thanks, it worked.

Reply:
That's great. Well done!